src/expr/src/relation/func.rs

```diff
@@ -418,12 +418,17 @@ where
         })
 }
 
-fn string_agg<'a, I>(datums: I, temp_storage: &'a RowArena) -> Datum<'a>
+fn string_agg<'a, I>(
+    datums: I,
+    temp_storage: &'a RowArena,
+    order_by: &Vec<ColumnOrder>,
```

Comment:
This can probably be a `&[ColumnOrder]`. Usually clippy complains about that; I wonder why not this time.

Reply:
Pretty sure we disabled that one.